File: 58-length-of-last-word/length-of-last-word.java
class Solution {
    public int lengthOfLastWord(String s) {
        int count=0;
        for(int i=s.length()-1;i>=0;i--){
            if(s.charAt(i)==' ' && count==0){
                continue;
            }
            if(s.charAt(i)==' '){
                break;
            }
            count++;
        }
        return count;
    }
}

File: 58-length-of-last-word/length-of-last-word.py
class Solution(object):
    def lengthOfLastWord(self, s):
        """
        :type s: str
        :rtype: int
        """
        count=0
        for i in range(len(s),0,-1):
            if(s[i-1]==' ' and count==0):
                continue;
            if(s[i-1]==' '):
                break
            count+=1
        return count
        